packages/components/pie-button/test/visual/pie-button.spec.ts

```diff
@@ -23,10 +23,17 @@ const props: PropObject = {
     isFullWidth: [true, false],
     disabled: [true, false],
     isLoading: [true, false],
+    iconPlacement: [undefined, ...iconPlacements],
 };
 
+// TODO: Currently setting the slot to use a straight up SVG
```

Comment:
Might be worth checking, but I would think you could use `pie-icons-webc` now for this if you wanted to and replace the copy of the SVG.

Reply:
I tried to use the webc icons package but I got the same error mentioned in the TODO so I assume it isn't fixed yet.